CSS/style.css

```diff
@@ -56,3 +56,56 @@ img {
     box-sizing: inherit;
   }
 
+body{
+  font-family: 'Lato';
+  font-style: normal;
+  font-size: 16px;
+  line-height: 1.2;
```

Comment:
Don't you need measurement units here?

Reply:
What do you mean? If you talk about lineheight, it was written so in the video tutorial. The value can be a number without naming.

Reply:
Yep, I'm about line-height. Will it work properly? If you set it to simple number, without units, what's the real, computed value? I guess, browser will calculate it in pixels, but not sure. You can check it in devtools (Ctrl + shift + C and click on the element having this line-height -> find "Computed" styles tab and check the value)

The point is: you need to know what each line in your code does